roles/ovirt-host-deploy-libvirt-guests/meta/main.yml

```diff
@@ -0,0 +1,20 @@
+galaxy_info:
+  author: Petr Kotas
+  description: Role to manage libvirt-guests rules on oVirt hypervisor.
+  company: Red Hat, Inc.
+
+  license: Apache License 2.0
+
+  min_ansible_version: 2.3
+
+  platforms:
+  - name: EL
+    versions:
+    - 7
+  - name: Fedora
+    versions:
+    - 25
```

Comment:
What about 26? Why limit the platforms?

Reply:
Ondro, do we need always to specify platforms for Galaxy? If so then we will need to include FC26 and remove FC24 for all roles in master branch ...
@machacekondra

Reply:
I will wait for the resolution.

Reply:
No really, but we dont support vdsm on anything different then fc and el, no? So I think it's better if it explicitly said.

Reply:
@machacekondra, better in what aspect? Does it help with anything?

Reply:
So let's have there only CentOS 7, because our Fedora support will never be up-to-date

Reply:
It doesn't help with anything it's just a text generated in galaxy README, which I guess wouldn't be used by anyone, because this is just internal role.